### PR TITLE
Wire up SQS, store, etc. in main program

### PR DESCRIPTION
**Pull request by arielshaqed**

### CODE REVIEW

**Comment by arielshaqed** [comment]
Tested manually on an SQS topic that I created, uploading files to my S3 bucket, and verifying updated values on Postgres.

**Comment by arielshaqed** [comment]
Sorry... merged 3f2de86 and 6ffc4e6 into this.  So please ignore Dockerfile, it was already reviewed in #6 .

File: cmd/terminus/main.go [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+var runCmd = &cobra.Command{
+	Use:     "run",
+	Short:   "Start the Terminus server",
+	Example: "terminus run --sqs-name=terminus-queue --db-dsn=postgres:///",
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx := context.Background()
+		dbDriver, err := cmd.Flags().GetString("db-driver")
+		DieOnErr(err)
+		dbDSN, err := cmd.Flags().GetString("db-dsn")
+		DieOnErr(err)
```
Comment (itai-david) [inline cmd/terminus/main.go:60]: This pattern returns multiple times. Maybe we should refactor into designated function:
```go
func GetStringFromFlagsOrDie(flag string) string {
  val, err := cmd.Flags().GetString(flag)
  DieOnErr(err)
  return val
}
```

And then:
```go
dbDriver := GetStringFromFlagsOrDie("db-driver")
dbDsn := GetStringFromFlagsOrDie("db-dsn")
```
etc

File: cmd/terminus/main.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+	rootCmd.AddCommand(runCmd)
+
+	runCmd.Flags().StringP("sqs-name", "q", "", "Name of topic on SQS with S3 events to process")
+	runCmd.MarkFlagRequired("sqs-name")
+
+	runCmd.Flags().String("db-driver", "pgx", "Database driver code")
+	runCmd.Flags().StringP("db-dsn", "d", "", "DSN to connect to database")
+	runCmd.MarkFlagRequired("db-dsn")
+
+	runCmd.Flags().StringP("pattern", "p", `^s3://[^/]*/user/([^/]*)/.*$`, "Regexp matching paths to track")
```
Comment (itai-david) [inline cmd/terminus/main.go:103]:
```suggestion
	runCmd.Flags().StringP("pattern", "p", `^s3://[^/]+/user/([^/]+)/.*$`, "Regexp matching paths to track")
```
I might be totally wrong, but it seems the original allows empty sequnces between `/` except for the 'user' literal. Is that on purpose?

**Review by itai-david — COMMENTED** [review]
2 small suggestions to consider. Other then that LGTM

**Comment by arielshaqed** [comment]
Continued on #9 due to my mixup :-(

**Pull request closed without merging**